### PR TITLE
docs: translation for basic plugin

### PR DESCRIPTION
**Pull request by cemremengu**
This PR is intended to provide an translation for `/basics/plugin.md` missing in #363 

I am not sure about the part

```
** Note: **

* `plugin.default.js` does not exists. Use `local` for dev environments.

* Use this feature only in the application layer. Do not use it in the framework layer.
```

### CODE REVIEW

File: docs/source/en/basics/plugin.md [hunk 1]
```diff
@@ -1 +1,176 @@
...
+
+```js
+// config / plugin.local.js
+exports.dev = {
+  enable: true,
+  package: 'egg-dev'
+};
+```
+
+This way, `npm i --production` in the production environment does not need to download the`egg-dev` package.
```
Comment (SunShinewyf) [inline docs/source/en/basics/plugin.md]: This way -> in the way

File: docs/source/en/basics/plugin.md [hunk 2]
```diff
@@ -1 +1,176 @@
...
+
+** Note: **
+
+* `plugin.default.js` does not exists. Use `local` for dev environments.
+
+* Use this feature only in the application layer. Do not use it in the framework layer.
+
+### Package name and path
+
+* The `package` is introduced in the `npm` require style which is the most common way to import
```
Comment (SunShinewyf) [inline docs/source/en/basics/plugin.md]: in the `npm` require style 改为 in the `npm` way ，require 感觉冗余

File: docs/source/en/basics/plugin.md [hunk 3]
```diff
@@ -1 +1,176 @@
...
+    database: 'test'
+  }
+};
+```
+
+Specific consolidation rules can be found in [Configuration] (. / Config.md).
+
+## Plugin list
+
+* Framework default built-in enterprise applications [Common plug-in](https://eggjs.org/zh-cn/plugins/):
```
Comment (SunShinewyf) [inline docs/source/en/basics/plugin.md]: built-in 是个形容词，在这里应该为动词才对 builds in

**Comment by cemremengu** [comment]
Ping @SunShinewyf

File: docs/source/en/basics/plugin.md [hunk 4]
```diff
@@ -1 +1,176 @@
...
+Each configuration item in `plugin.js` supports:
+
+* `{Boolean} enable` - Whether to enable this plugin, the default is true
+* `{String} package` -`npm` module name, plugin is imported via `npm` module
+* `{String} path` - The plugin's absolute path, mutually exclusive with package configuration
+* `{Array} env` - Only enable plugin in the specified runtime (environment), overriding the plugin's own configuration in `package.json`
+
+### Enabling/Disabling plugins
+
+The application does not need the package or path configuration when using the plugins built in the framework. You only need to specify whether they are enabled or not:
```
Comment (SunShinewyf) [inline docs/source/en/basics/plugin.md]: the framework -> the upper framework

**Comment by SunShinewyf** [comment]
把这个改了就可以合并了哈~

**Comment by cemremengu** [comment]
@SunShinewyf done

**Comment by SunShinewyf** [comment]
@atian25

**Pull request merged**